repairnator/repairnator-pipeline/src/main/java/fr/inria/spirals/repairnator/pipeline/Launcher.java

```diff
@@ -158,6 +158,11 @@ private JSAP defineArgs() throws JSAPException {
         opt2.setRequired(true);
         opt2.setDefault(repairTools);
         jsap.registerParameter(opt2);
+        
+        Switch sw1 = new Switch("skipDelete");
+        sw1.setLongFlag("skipDelete");
+        sw1.setDefault("false");
+        jsap.registerParameter(sw1);
```

Comment:
If this is really necessary, you should do this way:
```
// --skipDelete
jsap.registerParameter(LauncherUtils.defineArgSkipDelete());
```
Because this arg definition already exists.

Also, in the method `initConfig` of such class, you should do this:

`this.config.setSkipDelete(LauncherUtils.getArgSkipDelete(arguments));`

To set the arg value in `RepairnatorConfig`.

Reply:
Fixed

Reply:
Thank you, Benjamin. But I'm not sure it's necessary to change this class. Simon will probably check it before merging anyway.